modules/ROOT/pages/dataweave-runtime-functions.adoc

```diff
@@ -1,15 +1,44 @@
-= External Functions Available In DataWeave
+= External Functions Available to DataWeave
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:keywords: lookup, properties, functions
+:keywords: lookup, causedby, p, properties, functions
 
-In addition to the built-in DataWeave functions, Mule Runtime and some modules, connectors, and components for Mule apps provide functions that you can use in DataWeave scripts. The functions are specific to the module, connector, and components.
+In addition to the built-in DataWeave functions, Mule Runtime and some
+operations and components provide functions that you can use in
```

Comment:
Operations do not provide functions, modules, connectors and components (like batch) do.